pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -9,20 +9,30 @@ package apiserver
 
 import (
 	"context"
+	"encoding/json"
 	"io/ioutil"
+	"strconv"
 	"time"
 
 	log "github.com/cihub/seelog"
 	"github.com/ericchiang/k8s"
-	yaml "gopkg.in/yaml.v2"
 
+	"github.com/ericchiang/k8s/api/v1"
+
+	"fmt"
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/util/retry"
-	"github.com/ericchiang/k8s/api/v1"
 )
 
 var globalApiClient *APIClient
 
+const (
+	configMapDCAToken = "configmaptokendca"
+	defaultNamespace  = "default"
+	tokenTime         = "tokenTimestamp"
+	tokenTimeout      = 60
```

Comment:
This is specific to event collection, other application will require different freshness values (or event won't care). This should be passed as an argument